components/src/md3/empty-state/empty-state.tsx

```diff
@@ -76,7 +77,7 @@ export const EmptyState: React.FC<EmptyStateProps> = (props :EmptyStateProps) =>
         return Math.max(100, Math.min(200, iconSize));
     }, [iconSize]);
 
-    const getColor = useCallback((color: string | undefined): string => color || theme.colors.primary, [theme]);
+    const getColor = useCallback((color: string | undefined): string => color || theme.colors.outlineVariant, [theme]);
```

Comment:
We may want to leave a `// @TODO:` comment here mentioning that we will need to update this to theme.colors.disabled long term